src/utils/slpParser.ts

```diff
@@ -122,6 +126,10 @@ export class SlpParser extends EventEmitter {
     return this.frames;
   }
 
+  public getRollbackFrames(): RollbackFrames {
+    return this.rollbackCounter.get();
```

Comment:
This would need to be updated to make the 3 different calls to `rollbackCounter` to fit the shape of `RollbackFrames`.